editor/src/components/context-menu-items.ts

```diff
@@ -366,7 +366,27 @@ export function showReplaceComponentPicker(
 
 export const convert: ContextMenuItem<CanvasData> = {
   name: 'Swap Element With…',
-  shortcut: 'S',
+  shortcut: '',
+  enabled: (data) => {
+    return (
+      data.selectedViews.length > 0 &&
+      data.selectedViews.every((path) => {
+        const element = MetadataUtils.findElementByElementPath(data.jsxMetadata, path)
+        return (
+          element != null && isRight(element.element) && isJSXElementLike(element.element.value)
+        )
+      })
+    )
+  },
+  action: (data, _dispatch, _coord, event) => {
+    // FIXME Render prop support
```

Comment:
Render prop support is now possible, so you can follow a similar pattern to that used below in "Replace Element With"